samples/subsys/usb/console-next/src/main.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/drivers/uart.h>
+
+BUILD_ASSERT(DT_NODE_HAS_COMPAT(DT_CHOSEN(zephyr_console), zephyr_cdc_acm_uart),
+	     "Console device is not ACM CDC UART device");
+
+int main(void)
+{
+	const struct device *const dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_console));
+	uint32_t dtr = 0;
+
+	/* Poll if the DTR flag was set */
```

Comment:
is this to wait for the host to enumerate? may be a good idea to expand the comment I don't think it's hyper obvious

Reply:
We have it in all CDC ACM samples. It waits for the DTR flag to be set, which usually happens when an application on the host side opens the device. https://docs.zephyrproject.org/latest/connectivity/usb/device/usb_device.html#console-over-cdc-acm-uart